integration_test/third_party_apps_data/applications/active_directory_ds/metadata.yaml

```diff
@@ -88,32 +88,26 @@ expected_metrics:
   value_type: INT64
   kind: GAUGE
   monitored_resource: gce_instance
-  labels: {}
```

Comment:
Forgot that fsm_lib.py relies on labels always exist. Thus modifying fsm_lib.py from `labels=sorted(metric['labels'])` to `labels=sorted(metric.get('labels'))`